src/utils/LaunchDarkly/features.ts

```diff
@@ -53,4 +54,18 @@ const getVariableType = (variations: any[]) => {
     }
 
     return VariableType.json
+}
+
+const mapSDKVisibility = (clientSideAvailability: LDFeature['clientSideAvailability']) => {
+    if (!clientSideAvailability) {
+        return undefined
+    }
+
+    const { usingEnvironmentIds, usingMobileKey } = clientSideAvailability
```

Comment:
Is it possible to have the `clientSideAvailability` object exist, but both `usingEnvironmentIds` and `usingMobileKey` both be false? If so, then I would imagine server visibility should be set to true in that case?

Reply:
What jason said actually